docs/pages/cloud/faq.mdx

```diff
@@ -61,43 +62,101 @@ After [connecting](#how-can-i-access-the-tctl-admin-tool) `tctl` to Teleport Clo
 
-### Do you encrypt data at rest?
+## Do you encrypt data at rest?
 
 Each deployment is using at-rest encryption using AWS DynamoDB and S3 at-rest encryption
 for customer data including session recordings and user records.
+
+## Can I get a list of IP addresses to configure a firewall?
+
+We don't currently offer a list of addresses for firewall configuration, but intend to start offering this in early 2022.
+
+## Can I configure Teleport Cloud to only allow connections from a whitelist of IP addresses?
+
+No. We believe mTLS with strong identity is the best standard available for authenticating and access clients.
+We are also investigating additional and new features that can be used as a defense in depth strategy for use with Teleport Cloud in 2022.
+
+## Is IPv6 Supported for connections to Teleport Cloud?
+
+We don't currently support IPv6 connections to Teleport Cloud. 
+
+## Can I change the domain name of my Cloud instance after it's been created?
+
+We're currently researching whether this can be done, so please contact support.
+
+## Can I retrieve diagnostics from my hosted cluster?
+
+We currently don't expose any metrics interfaces for a tenant.
+
+For our own metrics collection, we're rolling out mTLS, so that only authorized clients may collect or scrape metrics from the running instances. 
+So we would need to extend our certificate management to external clients.
+
+Additionally, we would need to allow all components of a tenant to be individually addressable and reachable from outside our network (so a scraper can scrape the individual targets). 
+This could allow those individual components to be selectively attacked, if an adversary is able to address requests to specific targets within our network.
+
+## Is it possible to enable proxy recording mode?
+
+No, having the proxy man-in-the-middle connections to record the session is disabled for cloud customers.
+
+## Is there a status page available?
+
+[status.teleport.sh](https://status.teleport.sh)
+
+## Is there a way to download session recordings for easy playback?
+
+The ability to download recordings for offline viewing will be available in a future release.
+
+## I'm noticing high latency / terminal sessions are slow ?
+
+Please reach out to support@goteleport.com. Our latency design is still being rolled out, and needs to be opted into.
+
+## What is the maximum number of nodes a customer can connect to their cluster?
+
+If you plan on connecting more than 10,000 nodes or agents, please contact your account executive or customer support to ensure your tenant is properly scaled.
+
+## Are internal connections encrypted / authenticated?
+
+Teleport components communicate with themselves using mTLS, with a separate certificate authority for each tenant. Connections to AWS services, such as DynamoDB and
+S3 are done so using the in-transit and encryption at rest provided by AWS. Each tenant has it's own credentials that isolate it to interacting with only it's own data.
```

Comment:
- This should have a comma after "S3."
- I think we should replace "are done so" with "are established" or "are made."
- Can we replace "the in-transit and encryption at rest provided by AWS" with "the at-rest and in-transit encryption provided by AWS"? Or maybe, "encryption provided by AWS, both at rest and in transit"?
- "it's" should be "its" when possessive.